src/WebVaultClient.js

```diff
@@ -251,6 +251,7 @@ class WebVaultClient {
   async login(masterPassword) {
     await this.initFinished
     const login = await this.authService.logIn(this.email, masterPassword)
+    await this.sync()
```

Comment:
Ca ne me gêne pas de le mettre ici (ça empêchera juste de faire des login sans sync) par contre j'aurais préféré ne pas faire un await immédiat sur le sync. Le plus souvent les données sont déjà dans le store et il n'y a pas besoin de faire un sync avant de commencer à faire des choses.

Ce qu'on peut faire c'est un 
```suggestion
    this.loginSync = this.sync()
```

Et si quelqu'un veut s'assurer qu'il y a eu au moins un sync avant de commencer à faire quoi que ce soit, il pourra toujours faire un await dessus.

Reply:
(ou attendre l'événement sync plutôt que l'événement login)

Reply:
Ou peut-être mieux : Aller voir dans les store s'il y a des ciphers et domaines déjà en cache. Si oui on retourne sans l'await, sinon on fait un await sync

Reply:
```
const syncFinished = this.sync()
const hasData = await this.hasData()
if (! hasData) {
  await syncFinished
}
```

Je suis en train de voir pour le hasData